microlens/src/Lens/Micro.hs

```diff
@@ -764,7 +803,7 @@ You can also use it with e.g. '_Left' (and other 0-or-1 traversals) as a replace
 True
 -}
 has :: Getting Any s a -> s -> Bool
-has l = getAny #. foldMapOf l (\_ -> Any True)
```

Comment:
Small nit: this is the implementation from `lens`. I prefer to keep the `lens` definitions for simplicity so that users can easily tell that there's no difference.